pkg/users/password-dialogs.js

```diff
@@ -148,7 +148,10 @@ export function password_quality(password) {
                     }
                 })
                 .fail(function(ex) {
-                    reject(new Error(ex.message || _("Password is not acceptable")));
+                    if (force)
+                        reject(new Error(ex.message || _("Password is not acceptable")));
```

Comment:
... because here it can absolutely fail with a generic OS error.

Reply:
riight, I can probably check for some strings so it makes sure that the password was rejected, but on the other hand it feels like even if it fails in some buggy way (like `pwscore` fails because it misses some library or what now) we still should allow users to continue.